src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/ReleasesController.cs

```diff
@@ -159,5 +160,11 @@ private bool ReleaseExists(Guid id)
         {
             return _context.Releases.Any(e => e.Id == id);
         }
+
+        private int NextOrder(Release release)
+        {
+            var rel = _context.Releases.OrderByDescending(r => r.Order).FirstOrDefault(r => r.Publication.Id == release.PublicationId);
```

Comment:
Does the FirstOrDefault not return a default value? I.e. do you need the next line?

Reply:
Returns null - there's no orelse as per java sadly.